docs/inference/examples/margtime.rst

```diff
@@ -0,0 +1,53 @@
+######################################
+Marginalized time model
+######################################
+
+-------------------------------
+Example with GW150914
+-------------------------------
+
+The marginalized time model can handle a broad array of waveform models which
+it generates at full resolution like the ``gaussian_noise`` model. However,
+it is optimized to enable marginalization of time in addition to marginalization
+over sky location, polarization, and overall phase (valid if the waveform
+approximant is dominant mode only).
+
+This example demonstrates using the ``marginalized_time`` model with the
+``dynesty`` sampler in a configuration designed to run in couple minutes on a
+laptop. Actual sampling will only occur over the component masses and
+inclination. The remaining parameters are marginalized over.
+First, we create the following configuration file:
+
+.. dropdown:: Configuration File
+    :animate: fade-in-slide-down
+
+    .. literalinclude:: ../../../examples/inference/margtime/margtime.ini
+       :language: ini
+
+    :download:`Download <../../../examples/inference/margtime/margtime.ini>`
+
+For this example, we'll need to download gravitational-wave data for GW150914:
+
+.. literalinclude:: ../../../examples/inference/margtime/get.sh
+   :language: bash
+
+By setting the model name to ``marginalized_time`` we are using
+:py:class:`MarginalizedTime <pycbc.inference.models.marginalized_gaussian_noise.MarginalizedTime>`.
+
+Now run:
+
+.. literalinclude:: ../../../examples/inference/margtime/run.sh
+   :language: bash
+
+:download:`Download <../../../examples/inference/margtime/run.sh>`
+
+This will run the ``dynesty`` sampler. When it is done, you will have a file called
+``demarg_150914.hdf`` which contains the results. It should take just a few minutes to run.
```

Comment:
Add something here about how you are reconstructing the marginalized parameters after the inference is done. When I first looked at this, I missed that in the script, so was confused why the marginalized parameters were showing up in the corner plot.

Reply:
@cdcapano Ok,  I've updated the text to point this out.